brainstormforce/astra
# AST-3248 - improvement: Added a new filter 'astra_show_banner_video' to display/hide admin page banner YouTube video

### Description
- New filter for hide/show banner admin UI

### Screenshots
- Before - [Image 2023-06-29 at 6.01.40 PM](https://share.getcloudapp.com/X6u70gKD) 
- After adding filter - [Image 2023-06-29 at 6.02.14 PM](https://share.getcloudapp.com/ApugG7vx)

### Types of changes
- Bug fix

### How has this been tested?
- Check with filter add/remove case
- Check with whitelabel case
- By anyways adding above filter should hide the banner video

### Checklist:
